### ISSUE TITLE
Add Important Questions to Binary Search Section

### ISSUE THREAD

**Issue by thisisnitish** [neetcode-gh/leetcode#661]
I was revising Binary Search from [**neetcode.io**](neetcode.io) and then I realised that, some of the very important questions from Binary Search are missing. Below some of them are:

- Aggressive Cows - [https://www.spoj.com/problems/AGGRCOW/](https://www.spoj.com/problems/AGGRCOW/)
- InterviewBit Question Allocate Books - [https://www.interviewbit.com/problems/allocate-books/](https://www.interviewbit.com/problems/allocate-books/)
- Allocate minimum number of pages - [https://practice.geeksforgeeks.org/problems/allocate-minimum-number-of-pages0937/1#](https://practice.geeksforgeeks.org/problems/allocate-minimum-number-of-pages0937/1#)
- Painter's Partition Problem - [https://www.interviewbit.com/problems/painters-partition-problem/](https://www.interviewbit.com/problems/painters-partition-problem/)

I would like to see these questions in [**neetcode.io**](neetcode.io). I know these questions are not on LeetCode. But, hey, we need to learn the concepts right?? Irrespective of the platform we use.

Thanks

**Comment by Ahmad-A0** [comment]
I agree that these are essential questions to learn, but I think their leetcode counterparts should be added instead such as [1552. Magnetic Force Between Two Balls](https://leetcode.com/problems/magnetic-force-between-two-balls) in place of Aggressive Cows and [1011. Capacity To Ship Packages Within D Days](https://leetcode.com/problems/capacity-to-ship-packages-within-d-days/) in place of Allocate Books, etc. I only say this because of the way the repo is named and organised.
